test/AWSUtils.Tests/StorageTests/PersistenceGrainTests_AWSDynamoDBStore.cs

```diff
@@ -211,5 +214,15 @@ public void AWSDynamoDBStore_ConvertToFromStorageFormat_GrainReference_List()
             }
             Assert.Equal(initialState.Grain, convertedState.Grain);  // "Grain"
         }
+
+        private async Task<DynamoDBStorageProvider> InitDynamoDBTableStorageProvider(IProviderRuntime runtime, string storageName)
+        {
+            Dictionary<string, string> providerCfgProps = new Dictionary<string, string>();
+            var store = new DynamoDBStorageProvider();
+            providerCfgProps["DataConnectionString"] = TestDefaultConfiguration.DataConnectionString;
+            var cfg = new ProviderConfiguration(providerCfgProps, null);
+            await store.Init(storageName, runtime, cfg);
```

Comment:
have you tested this change? because we don't run this in CI

Reply:
Yes! tested against real dynamoDB